VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.8.204
+1.8.205
```

Comment:
Why isn't this 1.9?

Reply:
This is still a candidate for ONNX 1.9.0 release. Currently, we're testing the package on TestPyPi.